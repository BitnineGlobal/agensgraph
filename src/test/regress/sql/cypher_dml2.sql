CREATE GRAPH cypher_dml2;

CREATE VLABEL v1;

EXPLAIN ( analyze false, verbose true, costs false, buffers false, timing false )
MATCH (p:v1)
return max(collect(p.name)) as col;

MATCH (p:v1)
return max(collect(p.name)) as col;

MATCH (p:v1)
with collect(p.name) as col
RETURN max(col);

DROP GRAPH cypher_dml2 CASCADE;