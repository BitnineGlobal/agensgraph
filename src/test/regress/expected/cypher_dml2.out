CREATE GRAPH cypher_dml2;
CREATE VLABEL v1;
EXPLAIN ( analyze false, verbose true, costs false, buffers false, timing false )
MATCH (p:v1)
return max(collect(p.name)) as col;
ERROR:  aggregate function calls cannot be nested
LINE 3: return max(collect(p.name)) as col;
                   ^
MATCH (p:v1)
return max(collect(p.name)) as col;
ERROR:  aggregate function calls cannot be nested
LINE 2: return max(collect(p.name)) as col;
                   ^
MATCH (p:v1)
with collect(p.name) as col
RETURN max(col);
 max 
-----
 
(1 row)

DROP GRAPH cypher_dml2 CASCADE;
NOTICE:  drop cascades to 4 other objects
DETAIL:  drop cascades to sequence cypher_dml2.ag_label_seq
drop cascades to vlabel ag_vertex
drop cascades to elabel ag_edge
drop cascades to vlabel v1
